ui/perfherder/helpers.js

```diff
@@ -392,10 +392,7 @@ export const getGraphsURL = (
   // the otherwise rather useless signature hash to avoid having to fetch this
   // information from the server)
   if (phFrameworksWithRelatedBranches.includes(performanceFrameworkId)) {
-    const branches =
-      alertRepository === 'mozilla-beta'
-        ? ['autoland']
-        : thPerformanceBranches.filter(branch => branch !== alertRepository);
+    const branches = alertRepository === 'mozilla-beta' ? ['autoland'] : [];
```

Comment:
The line
`import { thPerformanceBranches } from '../helpers/constants';`
can be removed now.